setup.py

```diff
@@ -37,10 +40,11 @@
     install_requires=[
         'setuptools',
         'spyne>=2.12',
-        'lxml'
+        'lxml',
```

Comment:
Because the Soap11 service needs it. I moved it to the tests dependencies where it should be